distribution/tools/keystore-cli/src/test/java/org/elasticsearch/common/settings/KeyStoreWrapperTests.java

```diff
@@ -135,6 +135,18 @@ public void testDecryptKeyStoreWithWrongPassword() throws Exception {
         }
     }
 
+    public void testDecryptKeyStoreWithShortPasswordInFips() throws Exception {
+        assumeTrue("This should run only in FIPS mode", inFipsJvm());
+        KeyStoreWrapper keystore = KeyStoreWrapper.create();
+        keystore.save(env.configFile(), "alongenoughpassword".toCharArray());
```

Comment:
SHould we add the save when trying to save keystore with a short pwd?

Reply:
These both use the same code ( KeystoreWrapper#createCipher ) so I considered coverage sufficient. No objections to add one more test if you think it's useful though

Reply:
Leaving it up to you.
LGTM otherwise